### PR TITLE
Fix translation for Mermaid JS context menu label

### PR DESCRIPTION
**Pull request by Deepali1211**
## References

Fixes #https://github.com/jupyterlab/jupyterlab/issues/15174

## Code changes

Added the translator to translate the label

### CODE REVIEW

File: packages/mermaid-extension/src/index.ts [hunk 1]
```diff
@@ -65,13 +66,14 @@ const contextCommands: JupyterFrontEndPlugin<void> = {
   id: '@jupyterlab/mermaid-extension:context-commands',
   description: 'Provides context menu commands for mermaid diagrams.',
   autoStart: true,
-  requires: [IMermaidManager],
-  activate: (app: JupyterFrontEnd, mermaid: IMermaidManager) => {
+  requires: [IMermaidManager, ITranslator],
```
Comment (krassowski) [inline packages/mermaid-extension/src/index.ts:69]: Thank you for opening a PR quickly! Could you make `ITranslator` optional and fallback to `nullTranslator` to allow the plugin to be used with translator service disabled? For example see:

https://github.com/jupyterlab/jupyterlab/blob/bd46c76046671ae375b3ddd207ca441841e57123/packages/tooltip-extension/src/index.ts#L45-L51

Comment (Deepali1211) [reply packages/mermaid-extension/src/index.ts]: Done.

**Comment by krassowski** [comment]
Thank you! Could you also run `jlpm run integrity` locally and commit changes to reflect the new dependency to fix the [integrity check](https://github.com/jupyterlab/jupyterlab/actions/runs/6349236035/job/17247116668?pr=15178)?

**Comment by Deepali1211** [comment]
Integrity checks have passed now, but the Visual regression tests didn't succeed as the operation got canceled.

**Comment by krassowski** [comment]
No worries, I re-triggered the visual regression tests (it seems GitHub CI is slower this week and some jobs time out).

**Review by krassowski — APPROVED** [review]
Thank you @Deepali1211!

**Pull request merged**